### PR TITLE
Version bump and a couple of tslint errors fixed.

### PR DESCRIPTION
**Pull request by dlebu**
Non related, but all the code from `browser-workflow` is in `master` now. We will have all our PRs against `master` going forward.

### CODE REVIEW

**Comment by dlebu** [comment]
@guillaumejenkins @digeff @MSLaguana

File: src/debugger/cordovaDebugAdapter.ts [hunk 1]
```diff
@@ -411,18 +411,18 @@ export class CordovaDebugAdapter extends WebKitDebugAdapter {
     }
 
     private launchSimulate(launchArgs: ICordovaLaunchRequestArgs, generator: TelemetryGenerator): Q.Promise<void> {
-        let simulateTelemetryPropts = {
+        let simulateTelemetryPropts: any = {
```
Comment (MSLaguana) [inline src/debugger/cordovaDebugAdapter.ts]: I'd prefer if this was made into a proper interface to document the expected / allowed arguments.

Comment (dlebu) [inline src/debugger/cordovaDebugAdapter.ts]: Fixed.

**Comment by MSLaguana** [comment]
Looks good to me.

**Comment by dlebu** [comment]
Thanks @MSLaguana. Merging to master.

**Pull request merged**